src/components/Favicon/Favicon.tsx

```diff
@@ -1,20 +1,62 @@
 /* eslint-disable new-cap */
 import { Paper } from '@mui/material';
 import React from 'react';
-interface FaviconProps {
-  /* The link displayed in the UI */
-  domain: string;
+export interface FaviconProps {
+  /**
+   * Ideally only root domains should be given but most URLs will work.
+   *
+   * @example google.com
+   */
+  rootDomain: string;
 }
 
-/* A component to display the link to the article and its publishing date */
-const Favicon: React.FC<FaviconProps> = ({ domain }) => {
+/**
+ * A component to display the link to the article and its publishing date
+ *
+ * @return {React.FC}
+ */
+const Favicon: React.FC<FaviconProps> = ({ rootDomain }) => {
+  /**
+   * Removes the protocol, subdomain and folders (if any) from a given URL to get only the root domain.
+   *
+   * @example
+   * https://topics.factiverse.no/home gets turned into factiverse.no
+   *
+   * @param {string} url
+   * @return {string} The root domain
+   */
+  const getRootDomain = (url: string) => {
+    // TODO: This function fails if there are multiple domain names but no subdomain, is there a better way than including common exceptions?
+    // Special cases
+    if (url.includes('bbc.co.uk')) return 'bbc.co.uk';
```

Comment:
I don't think this is an issue. If the site is malicious with the URL evil.com/bbc.co.uk it will return bbc.co.uk and the BBC favicon would be displayed.